rcjaRegistration/invoices/views.py

```diff
@@ -55,6 +55,32 @@ def details(request, invoiceID):
 
     return render(request, 'invoices/details.html', context)
 
+@login_required
+def paypal(request, invoiceID):
+    # Get invoice
+    invoice = get_object_or_404(Invoice, pk=invoiceID)
+
+    # Check permissions
+    mentor = mentorInvoicePermissions(request, invoice)
+    if not mentor:
+        raise PermissionDenied("You do not have permission to view this invoice")
+
+    # Check paypal email is set for this state
+    if not invoice.paypalAvailable():
+        return HttpResponseForbidden('PayPal not enabled for this invoice')
+
+    # Set invoiced date
+    if mentor and invoice.invoicedDate is None:
```

Comment:
``if mentor`` here is redundant as mentor is checked for earlier in the control flow anyway

Reply:
Yes, but it was copied from a similar view where it isn't redundant and this makes the logic and reasoning more explicit.